python-sdk/tests/test_pps.py

```diff
@@ -124,6 +124,27 @@ def test_datums(client: TestClient, default_project: bool):
         with pytest.raises(grpc.RpcError, match=error_match):
             client.pps.restart_datum(job=job)
 
+    @staticmethod
+    def test_create_datum_client(client: TestClient):
```

Comment:
please update the name of this test to reflect the new name of the method being tested

Reply:
Good catch